.docs/content/2.guide/how-to-troubleshoot

```diff
@@ -0,0 +1,81 @@
+# How to troubleshoot
+
+Things can go wrong. As a developper, you know this already. But what to do when something goes wrong in ArmoniK ?
+
+This section will give you tips to check and fix problems for those times when it just won’t work and you do not know why.
+___
+## Your deployment failed or is malfunctioning
+
+First, note that if you deployment is local, MongoDB and Redis certifcates are only available for 7 days. Past this period, your deployment might encounter issues.
+Our suggestion is to destroy your deployment and redeploy.
+
+To check if your deployment is ok, you can enter the command:
+
+```bash [shell]
+kubectl get po
+```
+
+This line allows you to check the status of all the pods. If every one of them is «Running» or «Completed», your deployment is operational.
+___
+## Notes on Terraform
+If your deployment has an issue or you just want to stop it, we recommend using the following command:
+
+```bash [shell]
+sudo sysctemctl stop k3s.service
+```
+
+::alert{type="info"}
+We do **not** recommend using CTRL+C in the terminal to stop Terraform. The reason is that if you end Terraform this way, there will be no end message and your redeployment will have issues.
+::
```

Comment:
If Terraform CLI is interrupted (e.g. ctrl+c, SIGINT) during the apply phase (especially with a second interruption) then it will exit ungracefully (ex. Terraform state is corrupted). In that case, you will typically need to take manual repair actions before you can continue (which is usually very tedious.).